notifier/libnotify.go

```diff
@@ -1,28 +1,42 @@
 package notifier
 
 import (
-	"os/exec"
 	"sync"
 
 	log "github.com/sirupsen/logrus"
+	libnotify "github.com/menefotto/go-libnotify"
```

Comment:
Interesting that formatting didn't sort the imports for you, when I open the file locally it swaps line 6 and 7 - in any case, could you swap them too please?

Reply:
Formatting? What formatting? :P 
My Vim doesn't have any Go-specific features enabled, but I can do it manually instead. :)

Reply:
Ah I see 😄 One of the beauties of Go is that it comes with code formatting tool, so it eliminates any discussions about how code should be formatted because there is only one source of truth 🙂 I thought I have it as part of the Makefile, but no, it's just integrated in my own editor 😄

Reply:
Sorted imports in bc8be3b, `gofmt` seems to be content now :)